src/uct/api/uct.h

```diff
@@ -437,6 +442,23 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ * 
+ * These flags control completion semantic for @ref uct_ep_flush and
+ * @ref uct_iface_flush functions.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL  = UCS_BIT(0), /**< Guarantees flush of local
+                                             outstanding operations, the buffers
```

Comment:
currently it also guarantees data is delivered to remote peer (but maybe not to application), not just that buffers can be reused. Maybe 'LOCAL' is confusing, it better be UCT_FLUSH_FLAG_OUTSTANDING or UCT_FLUSH_FLAG_DELIVER ?
@shamisp what do you think?